[1m============================= test session starts ==============================[0m
platform linux -- Python 3.10.12, pytest-6.2.5, py-1.10.0, pluggy-0.13.0
cachedir: /home/jaay/Agnirath/Agnirath_LVS_Strategy/aarush_ros/build/gps_package/.pytest_cache
rootdir: /home/jaay/Agnirath/Agnirath_LVS_Strategy/aarush_ros/src/gps_package
plugins: ament-xmllint-0.14.1, ament-copyright-0.14.1, ament-flake8-0.14.1, launch-testing-2.0.2, ament-pep257-0.14.1, launch-testing-ros-0.24.0, ament-lint-0.14.1, colcon-core-0.12.1, cov-3.0.0
[1mcollecting ... [0m[1m
collected 3 items                                                              [0m

test/test_copyright.py [33ms[0m[32m                                                 [ 33%][0m
test/test_flake8.py [31mF[0m[31m                                                    [ 66%][0m
test/test_pep257.py [32m.[0m[31m                                                    [100%][0m

=================================== FAILURES ===================================
[31m[1m_________________________________ test_flake8 __________________________________[0m
[1m[31mtest/test_flake8.py[0m:23: in test_flake8
    [94massert[39;49;00m rc == [94m0[39;49;00m, \
[1m[31mE   AssertionError: Found 3 code style errors / warnings:[0m
[1m[31mE     ./gps_package/gps_node copy.py:59:77: E203 whitespace before ':'[0m
[1m[31mE     ./gps_package/gps_node.py:55:77: E203 whitespace before ':'[0m
[1m[31mE     ./gps_package/gps_node.py:104:1: E305 expected 2 blank lines after class or function definition, found 1[0m
[1m[31mE   assert 1 == 0[0m
----------------------------- Captured stdout call -----------------------------

./gps_package/gps_node copy.py:59:77: E203 whitespace before ':'
                    self.received_data = self.received_data[sentence_end + 2 :]
                                                                            ^

./gps_package/gps_node.py:55:77: E203 whitespace before ':'
                    self.received_data = self.received_data[sentence_end + 2 :]
                                                                            ^

./gps_package/gps_node.py:104:1: E305 expected 2 blank lines after class or function definition, found 1
if __name__ == "__main__":
^

2     E203 whitespace before ':'
1     E305 expected 2 blank lines after class or function definition, found 1

7 files checked
3 errors

'E'-type errors: 3

Checked files:

* ./setup.py
* ./test/test_flake8.py
* ./test/test_pep257.py
* ./test/test_copyright.py
* ./gps_package/__init__.py
* ./gps_package/gps_node copy.py
* ./gps_package/gps_node.py
----------------------------- Captured stderr call -----------------------------
option --max-complexity: please update from optparse string `type=` to argparse callable `type=` -- this will be an error in the future
- generated xml file: /home/jaay/Agnirath/Agnirath_LVS_Strategy/aarush_ros/build/gps_package/pytest.xml -
=========================== short test summary info ============================
FAILED test/test_flake8.py::test_flake8 - AssertionError: Found 3 code style ...
[31m============== [31m[1m1 failed[0m, [32m1 passed[0m, [33m1 skipped[0m, [33m2 warnings[0m[31m in 0.20s[0m[31m ==============[0m
[33m=============================== warnings summary ===============================[0m
test/test_flake8.py::test_flake8
test/test_flake8.py::test_flake8
  Warning: SelectableGroups dict interface is deprecated. Use select.

-- Docs: https://docs.pytest.org/en/stable/warnings.html
